main/src/main/scala/sbt/Defaults.scala

```diff
@@ -892,7 +901,7 @@ object Defaults extends BuildCommon {
       selectTests ~ options
     }
 
-  def distinctParser(exs: Set[String], raw: Boolean): Parser[Seq[String]] =
+  private def distinctParser(exs: Set[String], raw: Boolean): Parser[Seq[String]] =
```

Comment:
We missed this. This is a breaking change, see https://travis-ci.org/typesafehub/zinc/builds/79476798

```
[error] /home/travis/build/typesafehub/zinc/project/Scriptit.scala:56: method distinctParser in object Defaults cannot be accessed in object sbt.Defaults
[error]     Defaults.distinctParser(scriptitTests(scriptitBase, script).toSet, raw = false)
[error]              ^
```